capa/features/extractors/dnfile/helpers.py

```diff
@@ -41,56 +44,47 @@ def seek(self, offset: int) -> int:
         return self.offset
 
 
-class DnClass(object):
-    def __init__(self, token: int, namespace: str, classname: str):
-        self.token: int = token
-        self.namespace: str = namespace
-        self.classname: str = classname
+class DnType(object):
+    def __init__(self, token: int, class_: str, access: Optional[str] = None, namespace: str = "", member: str = ""):
```

Comment:
the order appears a little random here?

Reply:
good point - I'll move access to the end of the list to clean it up a little.